### PR TITLE
docs(Datepicker): Add cell customization

### PR DESCRIPTION
**Pull request by pompomon**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

Added examples for `calendarCell` and `calendarHeaderCell` customization using slots and children API

HeaderCell with Tooltip
image

Cell with Tooltip
image

### CODE REVIEW

File: packages/fluentui/docs/src/examples/components/Datepicker/Slots/DatepickerHeaderCellExample.shorthand.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import * as React from 'react';
+
+const DatepickerHeaderCellExample = () => {
+  return (
+    <Datepicker
+      today={new Date(2020, 8, 12, 0, 0, 0, 0)}
+      calendar={{
+        calendarHeaderCell: {
+          children: (ComponentType, props) => (
+            <Tooltip content={props['aria-label']} trigger={<ComponentType {...props} />} />
```
Comment (layershifter) [inline packages/fluentui/docs/src/examples/components/Datepicker/Slots/DatepickerHeaderCellExample.shorthand.tsx:11]: It looks that we will have `aria-label` on a `trigger` still, is it expected?

File: packages/fluentui/docs/src/examples/components/Datepicker/Slots/DatepickerCellExample.shorthand.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import * as React from 'react';
+
+const DatepickerCellExample = () => {
+  return (
+    <Datepicker
+      today={new Date(2020, 8, 12, 0, 0, 0, 0)}
+      calendar={{
+        calendarCell: {
+          children: (ComponentType, props) =>
+            props.isToday ? (
```
Comment (layershifter) [inline packages/fluentui/docs/src/examples/components/Datepicker/Slots/DatepickerCellExample.shorthand.tsx:11]: Is it really `isToday`? We don't use `is*` prefix in prop names...

**Review by layershifter — APPROVED** [review]

File: packages/fluentui/docs/src/examples/components/Datepicker/Slots/DatepickerHeaderCellExample.shorthand.tsx (hunk 1 above)
Comment (pompomon) [reply packages/fluentui/docs/src/examples/components/Datepicker/Slots/DatepickerHeaderCellExample.shorthand.tsx:11]: Yep :)

File: packages/fluentui/docs/src/examples/components/Datepicker/Slots/DatepickerCellExample.shorthand.tsx (hunk 2 above)
Comment (pompomon) [reply packages/fluentui/docs/src/examples/components/Datepicker/Slots/DatepickerCellExample.shorthand.tsx:11]: Ooops, will fix it with @rymeskar in a separate PR :D

**Pull request merged**